src/Azure.Functions.Cli/Kubernetes/KEDA/V2/Models/ScaledObjectV1Alpha1.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using Azure.Functions.Cli.Kubernetes.Models.Kubernetes;
+using Newtonsoft.Json;
+
+namespace Azure.Functions.Cli.Kubernetes.KEDA.V2.Models
+{
+    public class ScaledObjectV1Alpha1 : BaseKubernetesResource<ScaledObjectSpecV1Alpha1>
+    {
+        public override string ApiVersion => "keda.sh/v1alpha1";
+        public override string Kind => "ScaledObject";
+    }
+
+    public class ScaledObjectSpecV1Alpha1 : IKubernetesSpec
+    {
+        [JsonProperty("scaleTargetRef")]
+        public ScaledObjectScaleTargetRefV1Alpha1 ScaleTargetRef { get; set; }
+
```

Comment:
Reuse models that are common between V1 and V2. Please see https://github.com/tomkerkhove/azure-functions-core-tools/pull/2 - for the review and proposed changes.